### PR TITLE
Demos: Toggle map-page styling using switch

### LINKED ISSUE #8065: http://demos.jquerymobile.com/1.4.1/map-list-toggle/ list cannot slide on iPad latest
In the default demo there are a few items, but as you keep adding new items on the list using the 'show more' item at the bottom you find out that the list does not slide down to show unseen list items. It is limited to displaying only the top ones visible on screen.
This is verified on iPad, latest and latest OS, just updated. Probably iPhone will show the same unexpected behavior.
Tested with page
http://demos.jquerymobile.com/1.4.1/map-list-toggle/
not sure if the same demo is available on newer versions of jquery mobile

### PR DESCRIPTION
**Pull request by kakul**
Fixes gh-8065

### CODE REVIEW

**Comment by kakul** [comment]
@arschmitz I've made the changes.

File: demos/map-list-toggle/index.php [hunk 1]
```diff
@@ -8,8 +8,8 @@
 	<style>
 	.segmented-control { text-align:center; }
 	.segmented-control .ui-controlgroup { display:inline; margin:.2em 0px; }
-
-	#map-page { width:100%; height:100%; }
+	.map-page { width:100%; height:100%; }
```
Comment (arschmitz) [inline demos/map-list-toggle/index.php]: can you adjust this based on the css style guide please http://contribute.jquery.org/style-guide/css/

File: demos/map-list-toggle/index.php [hunk 2]
```diff
@@ -31,23 +31,22 @@
...
-	    });
+		var map = $( "#map-canvas" ),
+		    list = $( "#list-canvas" ),
+		    page = $( this );
 
-	    $listSwitch.on( "click", function( e ){
-	       	$list.show();
-			$map.hide();
-	    });
+		$( "#list-switch,#map-switch" ).on( "click", function( event ){
```
Comment (arschmitz) [inline demos/map-list-toggle/index.php]: missing space after comma

Comment (arschmitz) [inline demos/map-list-toggle/index.php]: you are not using event so remove it

File: demos/map-list-toggle/index.php (hunk 1 above)
Comment (kakul) [inline demos/map-list-toggle/index.php]: Oh okay i'll make the changes.

File: demos/map-list-toggle/index.php (hunk 2 above)
Comment (kakul) [inline demos/map-list-toggle/index.php]: I'll fix this.

**Pull request closed without merging**